### PR TITLE
Advanced install

### PR DESCRIPTION
**Pull request by hubvu**
- moved the air gap and declarative documentation under a "Advanced" section
- minor formatting + grammer fixes.

### CODE REVIEW

File: docs/install/DigitalOcean/_index.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 weight: 60
 description: >
-  Ondat deployment guides for Digital Ocean
+  Ondat deployment guides for DigitalOcean
 ---
 
 ## Overview
 
-If you are running kubernetes in Digital Ocean there are a number of options on how you run your kubernetes cluster.  Please see the following datasheets and blogs for more information about some of the use-cases of running in Digital Ocean
+If you are running kubernetes in Digital Ocean there are a number of options on how you run your kubernetes cluster. Please see the following datasheets and blogs for more information about some of the use-cases of running in Digital Ocean
```
Comment (Arau) [inline docs/install/DigitalOcean/_index.md:11]: Missing a `.` at the end.

File: docs/install/GCP/_index.md [hunk 2]
```diff
@@ -3,12 +3,12 @@ title: "GCP"
...
 weight: 20
 description: >
-  Ondat deployment guides for GCP
+  Ondat deployment guides for GCP.
 ---
 
 ## Overview
 
-If you are running kubernetes in GCP there are a number of options on how you run your kubernetes cluster.  Please see the following datasheets and blogs for more information about some of the use-cases of running in GCP
+If you are running kubernetes in GCP there are a number of options on how you run your kubernetes cluster. Please see the following datasheets and blogs for more information about some of the use-cases of running in GCP
```
Comment (Arau) [inline docs/install/GCP/_index.md:11]: Missing a `.` at the end.

File: docs/install/_index.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ description: >
   Ondat deployment guides for supported container orchestrators
 ---
 
-The following guides will help you get Ondat up and running on a variety of different environments.  If at any time you need help, please dont hesitate to contacts us on [Slack](https://storageos.slack.com/ssb/redirect#/shared-invite/email) or [book in a call](https://www.ondat.io/request-demo) where one of our solution architects can talk through your particular scenario.
+The following guides will help you get Ondat up and running on a variety of different environments. If at any time you need help, please dont hesitate to contacts us on [Slack](https://storageos.slack.com/ssb/redirect#/shared-invite/email) or [book in a call](https://www.ondat.io/request-demo) where one of our solution architects can talk through your particular scenario.
```
Comment (Arau) [inline docs/install/_index.md:9]: s/dont/don't/

??

File: docs/install/DigitalOcean/_index.md (hunk 1 above)
Comment (hubvu) [reply docs/install/DigitalOcean/_index.md]: fixed!

File: docs/install/GCP/_index.md (hunk 2 above)
Comment (hubvu) [reply docs/install/GCP/_index.md]: fixed!

File: docs/install/_index.md (hunk 3 above)
Comment (hubvu) [reply docs/install/_index.md]: fixed!

**Review by cannischan — APPROVED** [review]

**Pull request merged**